core-ui/src/components/Extensibility/ExtensibilityDetails.js

```diff
@@ -33,17 +31,17 @@ export const ExtensibilityDetailsCore = ({ resMetaData }) => {
   const breadcrumbs = [
     {
       name: t('name', {
-        defaultValue: resMetaData.resource?.kind,
+        defaultValue: prettifyNamePlural(resMetaData.resource.path),
```

Comment:
Maybe use `prettifyKind` with `pluralize` here? It handles the spaces better.
<img width="560" alt="image" src="https://user-images.githubusercontent.com/10504661/173766720-1fb61441-6a3c-4834-995f-60839735ff9a.png">